src/ansys/aedt/toolkits/magnet_segmentation/backend/workflows/aedt.py

```diff
@@ -100,25 +100,20 @@ def segmentation(self):
         if not self.properties.is_skewed:
             magnets = self.aedtapp.modeler.get_objects_by_material(self.properties.magnets_material)
             if self.properties.rotor_material == self.properties.stator_material:
-                if self.properties.motor_type == "IPM":
-                    for obj in self.aedtapp.modeler.get_objects_by_material(self.properties.rotor_material):
-                        if [i for i in magnets if i in self.aedtapp.modeler.objects_in_bounding_box(obj.bounding_box)]:
-                            rotor = obj
-                elif self.properties.motor_type == "SPM":
-                    for obj in self.aedtapp.modeler.get_objects_by_material(self.properties.rotor_material):
-                        if not [
-                            i for i in magnets if i in self.aedtapp.modeler.objects_in_bounding_box(obj.bounding_box)
-                        ]:
-                            rotor = obj
-                else:
-                    raise RuntimeError(f"Motor type {self.properties.motor_type} is not yet handled.")
```

Comment:
Don't you want to keep testing the motor type and raise a runtime error if it isn't IPM or SPM ?

Reply:
@SMoraisAnsys this makes sense but I implemented a change that is able to recognize stator and rotor object regardless the motor type. The original reason why I implemented the motor type was to be able to detect both object but I found that there's a specific SPM motor type that doesn't work with this check so I had to find a new way.

Reply:
@SMoraisAnsys in fact, I'm thinking to remove the motor_type